### PR TITLE
Multiple UI changes

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Fixing the inner-wrap outside of the actual app and fixing the userlist in the system console.

### CODE REVIEW

File: webapp/components/sidebar.jsx [hunk 1]
```diff
@@ -184,10 +184,15 @@ export default class Sidebar extends React.Component {
...
     }
+    hideSidebar() {
+        $('.inner-wrap').toggleClass('move--right');
+        $('.sidebar--left').toggleClass('move--right');
+    }
     onChange() {
         this.setState(this.getStateFromStores());
     }
     updateTitle() {
+        this.hideSidebar();
```
Comment (crspeller) [inline webapp/components/sidebar.jsx]: This doesn't belong here. Update title triggers hide sidebar?

**Comment by asaadmahmoodspin** [comment]
Hey guys, I removed the sidebar thing.

**Comment by jwilander** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**